behind-the-veil-siteroot/imports/api/methods/chats.js

```diff
@@ -0,0 +1,70 @@
+// NOTE: for the bookings details page, when they click 'send message' button
+// we have to check if the chat exists - if yes, return the id, if no - create a new chat object
+
+/**
+ * File Description: Chat database entity
+ * File version: 1.0
+ * Contributors: Vicky
+ */
+import { ChatCollection } from "../collections/chats";
+
+Meteor.methods({
+    "create_chat": function (brideUsername, artistUsername, chatUpdatedDate, chatLastMessage, readByBride, readByArtist) {
+        /**
+         * Adds a new booking to the database. Keep in mind this is an async met
```

Comment:
tiny thing, is this comment supposed to say "adds a new chat" instead of "booking"?

Reply:
Oops yeah it is, I must've missed changing it when copying methods from bookings - just fixed and pushed it